README.md

```diff
@@ -116,3 +116,15 @@ You may be able to get away without thinking about all this if your project spec
 Error suggestions may be triggered from some underlined compiler warnings. There
 is no visual indication, currently this will basically be for 'import' warnings,
 and can be triggered by 'alt-enter' (PureScript: Show Quick Fixes).
+
+## Hacking on atom-ide-purescript
+
+Install dependencies
+  • `bower install`
+  • `npm install`
+
+Build: `pulp build`
```

Comment:
So `npm install` will also trigger `bower install` or are you saying that when you run `npm run bundle` that will do the bower/npm installation?